app/views/partials/personal_key/_key.slim

```diff
@@ -1,13 +1,13 @@
-#recovery-code(class="col-12 border-box mt4 mb3 py2 px4 fs-20p sans-serif\
+#recovery-code(class="col-12 border-box mt4 mb3 py2 px2 sm-px4 fs-20p sans-serif\
                       border border-dashed border-red rounded-xl relative clearfix")
   = image_tag asset_url('scissors.svg'), width: 24,
     class: 'absolute', style: 'top:-14px;left:-5px'
   p.bold.center.mt1
     = image_tag(asset_url('p-key.svg'), width: 36, class: 'align-middle mr1')
     = t('users.recovery_code.header')
-  .navy.monospace
+  .my4.px0.sm-px1.py2.center.border-box.border.border-teal.bw2.rounded-md.separator-text
     - code.split(' ').each do |word|
-      p.bold.mb0.h3(data-recovery='word')
+      .inline.h3.bold.navy.monospace(data-recovery='word')
```

Comment:
@el-mapache It was 100% to appease rubocop/line length since the containing div already had a record breaking 11 classes on it.